README.md

```diff
@@ -45,6 +45,20 @@ Usually this script is not run after installation but could be used with some mo
 
 This script was originally contributed by [@Landrash](https://github.com/landrash).
 
+### Install MariaDB *(install_mariadb.sh)*
+This script installs MariaDB and it's dependencies for use with the [recorder](https://home-assistant.io/components/recorder/) component in Home Assistant.
```

Comment:
Maybe add note that this uses "MySql" connection from recorder?
Could be confusing for some.

Reply:
Submitting a PR to fix the documentation since it's lacking in that department.

Reply:
Addressed in https://github.com/home-assistant/home-assistant.github.io/pull/3484 .